images/build/cross/variants.yaml

```diff
@@ -8,7 +8,6 @@ variants:
     GO_MAJOR_VERSION: '1.21'
     OS_CODENAME: 'bullseye'
     REVISION: '0'
-    PROTOBUF_VERSION: '3.19.4'
```

Comment:
shouldn't we keep this version specified per release branch, to avoid picking up latest protoc across all build images for all release branches unintentionally?

Reply:
Yeah I agree, let's revert that change for now in https://github.com/kubernetes/release/pull/3196 and attempt an update in v1.29 if required.